introduction.md

```diff
@@ -1,14 +1,18 @@
-Introduction
-------------
+## Introduction
+
 ### Background
 
-<p class="ednote">This section is to be developed by the WCAG2ICT Task Force.</p>
+A W3C [Working Group Note](http://www.w3.org/2005/10/Process-20051014/tr#WGNote) on the topic of applying WCAG 2.0 to non-web content was approved in September of 2013.  That Working Group Note, [Guidance on Applying WCAG 2.0 to Non-Web Information and Communications Technologies (WCAG2ICT)](https://www.w3.org/TR/2013/NOTE-wcag2ict-20130905/) describes how WCAG 2.0 can be applied to ICT, specifically to non-web documents and software.  WCAG21CT is organized by sections to mirror those found in WCAG: Perceivable, Operable, Understandable, and Robust.  WCAG2ICT clarifies when and how WCAG success criteria should be applied to non-web documents and software.  Some success criteria are applicable without modification, and some success criteria are applicable with edits and/or as noted.  Glossary terms were also reviewed for applicability as written.  Level AAA Success Criteria were not addressed in the 2013 WCAG2ICT Working Group Note.
```

Comment:
Suggest replacing "with edits and/or as noted" with "with edits and/or notes".

Reply:
@bruce-usab I've incorporated these and the changes suggested by Olivia into your branch.